### DISCUSSION TITLE
Mapping EPUB Media-Overlays: style information?

### DISCUSSION THREAD

**Discussion by llemeurfr** [readium/guided-navigation#11] [General]
Guided-navigation is aimed as being a mapping of EPUB Media-Overlays. Media Overlays authors can provide style information, se https://www.w3.org/TR/epub/#example-associating-style-information-with-the-currently-playing-epub-content-document. 

Should we provide the equivalent during the mapping of EPUB Media Overlays to this json representation? If yes should we use webpub-manifest metadata (as a mapping from EPUB package metadata)?

**Comment by HadrienGardeur** [comment]
This is currently handled in the EPUB profile at https://github.com/readium/webpub-manifest/blob/master/profiles/epub.md#mediaoverlay

I think we'll eventually need an EPUB parsing documentation that will cover that, but I wouldn't add these property to the Guided Navigation document itself.
